tox.ini

```diff
@@ -1,32 +1,32 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,py36,pypy,pypy3,coverage,docs
+    py27,py34,py35,py36,pypy,pypy3
```

Comment:
Because the doctests are now run in all environments, and the `coverage` environment here is destructive and can't be run correctly in parallel with `detox`.